.travis.yml

```diff
@@ -0,0 +1,30 @@
+language: python
+python:
+  - 3.4
+  - 3.5
+  - 3.6
+
+cache:
+  pip: true
+
+before_install:
+  # Use pip 9, so that setuptools can be downgraded.
+  - pip install pip==9 setuptools==17
```

Comment:
I am pretty sure this (and comment above) isnt necessary for this repo; I believe you copied that from coala/coala , and it is there to test a specific type of problem that doesnt need to be tested here.